content/docs/command-reference/add.md

```diff
@@ -144,6 +144,11 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
+- `--glob` - searches for pathnames matching the patterns specified by the
```

Comment:
searches -> adds files ... (this is the primary function of the `dvc add` command)

Reply:
the idea here that people should be able to find it ... I would imagine the would look like (dvc add files matching patterns ... add directories matching ... etc)

Reply:
Yes. I agree. I have updated it to better fit the functionality of dvc.